src/libcore/intrinsics.rs

```diff
@@ -962,6 +962,33 @@ extern "rust-intrinsic" {
     /// value is not necessarily valid to be used to actually access memory.
     pub fn arith_offset<T>(dst: *const T, offset: isize) -> *const T;
 
+    /// Equivalent to the appropriate `llvm.memcpy.p0i8.p0i8.*` intrinsic, with
```

Comment:
Is this maybe a copy/paste typo? Should this mention that it's the atomic version?

Reply:
yep, I'll fix that.